src/js/api/api.js

```diff
@@ -250,12 +264,21 @@ define([
             return _instream;
         };
         _this.destroyPlayer = function () {
```

Comment:
I think your change removes the necessity for a "remove" separate from "destroy"

We can then delete this function and jam its logic into api.remove(). This allows us to delete the line _api.destroyPlayer() from globalApi's _removePlayer!